corehq/apps/users/models.py

```diff
@@ -2686,6 +2687,8 @@ class Invitation(models.Model):
     profile = models.ForeignKey("custom_data_fields.CustomDataFieldsProfile",
                                 on_delete=models.SET_NULL, null=True)
     custom_user_data = models.JSONField(default=dict)
+    tableau_role = models.CharField(max_length=32, choices=TABLEAU_ROLES, null=True)
+    tableau_group_ids = ArrayField(models.CharField(max_length=32), null=True)
```

Comment:
Does that limit  comes from here?
https://github.com/dimagi/commcare-hq/blob/4d72daac80b0d0ae34249f875f75e88e7f80811c/corehq/apps/reports/models.py#L251

Reply:
I saw the `TableauGroup` model that was eventually deleted in this [PR](https://github.com/dimagi/commcare-hq/pull/32379/files) had a 32 char limit so I based it off that

Reply:
I'm a little confused. Are we expecting the array of group ids to be strings like `HQ_TABLEAU_GROUP_NAME = 'HQ'` or some alphanumeric uuid's?

Reply:
If we know it will be id's in this format `4a30894a-dd66-4cbb-b14c-4f701135c9c5` of 36 chars, I would assume it would be safe to assume consistency and make the max 36 chars.

Reply:
Thanks for looking into this Addision, I should have checked myself before setting it to 32. Made the change t 36 here. [a07d982](https://github.com/dimagi/commcare-hq/pull/34445/commits/a07d982d24d511e4d46912fefea526f23172fda3)